composer.json

```
@@ -15,7 +15,7 @@
         "illuminate/filesystem": "^6.0 || ^7.0 || ^8.0",
         "illuminate/support": "^6.0 || ^7.0 || ^8.0",
         "illuminate/view": "^6.0 || ^7.0 || ^8.0",
-        "league/commonmark": "^1.5"
+        "league/commonmark": "^2.0.0"
```

To avoid BC breaks, it would be safer to allow either versions : 

```suggestion
        "league/commonmark": "^1.5 || ^2.0.0"
```

---

oh @yannoff using Laravel ! Insane!

---

> oh @yannoff using Laravel ! Insane!

@mcadet I just sense it's time to leave symfony before the wreck :stuck_out_tongue_closed_eyes: